cucumber/features/devLoginPup.feature

```diff
@@ -0,0 +1,9 @@
+Feature: TTA Smarthub HSES login
+ 
+    Scenario: Welcome page is displayed
+        When visiting https://tta-smarthub-dev.app.cloud.gov
+        Then we should see "Welcome to the TTA Smart Hub!" message
+    Scenario: Login is redirected to HSES
+        Given https://tta-smarthub-dev.app.cloud.gov
```

Comment:
Yes, we could consolidate, e.g. have either one `Given ` or one `When`.  This would eliminate the duplicated code in the step definition. We could also have functions for each code sections that can be called to be more DRY.